test-suite/js/test-stylesheet.css

```diff
@@ -0,0 +1,37 @@
+body {
+    margin: 1em;
```

Comment:
Our only existing .css file https://github.com/scala-js/scala-js/blob/master/assets/additional-doc-styles.css uses 2-space indentation. We should consistently keep that.